eePlumB/Z_PrepAOI/pySetup.R

```diff
@@ -0,0 +1,17 @@
+# a verbose version of this script exists in the pySetup.Rmd file
```

Comment:
Was this file part of the GEE uploading workflow you tried but decided to wait on? Do you want to still include this file here?

Reply:
Yup! This was an artifact! I'll pull that out when I update today! THANK YOU!

Reply:
file removed in 516d3a6